### ISSUE TITLE
Popup window does not go away if appeared due to button press on case

### ISSUE THREAD

**Issue by Mo-og** [steam3d/MagicPods-Windows#248]
If I press the pairing button on (AirPods Pro) case once (to see charging percentage on iPhone widgets) the MagicPods popup window appears and stays on screen forever until I manually close it or open and close the case.

**Comment by steam3d** [comment]
Hi. Why do you push the pairing button if you do not want to pair the AirPods?

https://user-images.githubusercontent.com/30986874/192000814-912105b9-2e5f-4a30-bfbc-a7b72cce8cf6.mp4

**Comment by Mo-og** [comment]
If I press and release it it acts as if I open the case. Probably no one even knows that button does that. I just said that if it was indeed pressed, the popup window will stay forever. So it might be better if it has some kind of timeout.

**Comment by steam3d** [comment]
Did you have a closed case? I tried on my AirPods Pro. When i pressed button nothing happens

**Comment by Mo-og** [comment]
Yes, case closed, both headphones inside. I tested it now and popup seems to go away after a minute+. Tested again after opening and closing the case and now if stays forever. Looks like it goes away if I do nothing with pc and then move mouse or click. But if I do something from start it stays (case was once opened, closed, then (after any time) pairing button was pressed). Not sure though.

**Comment by steam3d** [comment]
Well. It is complicate to figure out. I spent couple of hours to check. I can't detect that case is closed and sometime the iPhone show only right AirPod charge and case and sometimes both AirPods and case charge.

The MagicPods has ability to close popup window when you click or alt+tab to other app.

I will figure out what to do. I even did not know that button works when case closed.
